### PR TITLE
fix: use LAVAMOAT_UPDATE_TOKEN in attributions workflow

### PR DESCRIPTION
**Pull request by cryptodev-2s**
## **Description**

This PR replaces the use of `GITHUB_TOKEN` with `LAVAMOAT_UPDATE_TOKEN`, which has the necessary permissions to commit and comment on a PR. 

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/25731?quickstart=1)

## **Pre-merge author checklist**

- [ ] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: .github/workflows/update-attributions.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
             -f content='+1'
         env:
           COMMENT_ID: ${{ github.event.comment.id }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.LAVAMOAT_UPDATE_TOKEN }}
```
Comment (Gudahtt) [inline .github/workflows/update-attributions.yml:42]: Nit: We should create an issue or add a TODO comment for renaming this token

Comment (cryptodev-2s) [reply .github/workflows/update-attributions.yml:42]: I just created this issue https://github.com/MetaMask/metamask-extension/issues/25737

**Review by NEllusion — APPROVED** [review]
Looks good!

**Pull request merged**